src/webhint-theme/source/core/css/layouts.css

```diff
@@ -115,39 +115,39 @@ ol.layout {
 
 @media (min-width: 48em) {
 
-    [class*="layout--sidebar"] > .module {
+    [class*="layout--sidebar"] > .module--wrapper > .module {
         margin-top: 0;
     }
 
-    [class*="layout--sidebar"] > .module--secondary {
+    [class*="layout--sidebar"] > .module--wrapper > .module--secondary {
         width: 22rem;
     }
 
-    [class*="layout--sidebar"] > .module--primary {
+    [class*="layout--sidebar"] > .module--wrapper > .module--primary {
         width: calc(100% - 26rem);
```

Comment:
Same thing for the rest of these: just add back the class that was removed in addition to your updated class.

```
[class*="layout--sidebar"] > .module--primary,
[class*="layout--sidebar"] > .module--wrapper > .module--primary {
```